types/react-mathquill/tsconfig.json

```
@@ -0,0 +1,24 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictFunctionTypes": true,
+        "strictNullChecks": true,
+        "baseUrl": "../",
+        "typeRoots": [
+            "../"
+        ],
+        "jsx": "react",
+        "types": [],
+        "noEmit": true,
+        "forceConsistentCasingInFileNames": true
```

Do not forget to add this so that you can use the original name of the package.

```suggestion
        "forceConsistentCasingInFileNames": true,
        "paths": {
            "@edtr-io/mathquill": ["edtr-io__mathquill"]
        }
```